src/libs/actions/Policy.ts

```diff
@@ -3604,11 +3664,14 @@ function enablePolicyTaxes(policyID: string, enabled: boolean) {
                     },
                 },
             },
+            ...(shouldAddDefaultTaxRatesData ? taxRatesData.failureData ?? [] : []),
         ],
     };
 
     const parameters: EnablePolicyTaxesParams = {policyID, enabled};
-
+    if (shouldAddDefaultTaxRatesData) {
+        parameters.taxFields = CONST.DEFAULT_TAX.RATES;
```

Comment:
```suggestion
        parameters.taxFields = JSON.stringify(CONST.DEFAULT_TAX.RATES);
```
Instead of sending `taxFields` as object, let us stringify and send  as `string`

Reply:
This will fix the issue mentioned [here](https://github.com/Expensify/App/pull/38375#issuecomment-2012316101)

Reply:
I just updated to use ```parameters.taxFields = JSON.stringify(CONST.DEFAULT_TAX.RATES);```

Reply:
Below is the result:
https://github.com/Expensify/App/assets/161821005/820be212-8024-4d8e-bad7-bbe7d9551532